Epoch 0/99
----------
Traceback (most recent call last):
  File "train_PDAN.py", line 374, in <module>
    run([(rgb_model, 0, dataloaders, optimizer, lr_sched, args.comp_info)], criterion, num_epochs=int(args.epoch))
  File "train_PDAN.py", line 146, in run
    train_map, train_loss = train_step(model, gpu, optimizer, dataloader['train'], epoch)
  File "train_PDAN.py", line 220, in train_step
    outputs, loss, probs, err = run_network(model, data, gpu, epoch)
  File "train_PDAN.py", line 192, in run_network
    activation = model(inputs, mask_new)
  File "/home/areka/miniconda3/envs/PDAN/lib/python3.7/site-packages/torch/nn/modules/module.py", line 532, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/areka/pdan_da/PDAN.py", line 43, in forward
    self.summary = self.summarization_module(r_x)
  File "/home/areka/miniconda3/envs/PDAN/lib/python3.7/site-packages/torch/nn/modules/module.py", line 532, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/areka/pdan_da/PDAN.py", line 21, in forward
    inputs = torch.permute(inputs, (0, 2, 1)) # permute from (dim, T) to (T, dim)
AttributeError: module 'torch' has no attribute 'permute'